scss/_patterns_slider.scss

```diff
@@ -8,7 +8,6 @@
   }
 
   .p-slider__input {
-    height: 2.625rem;
```

Comment:
I couldn't see any reason that this had a specific size set, but maybe someone has a bit more insight...

Reply:
Maybe it was some magic number to get an older version of it look right?
It's fine to get rid of this I think.